### PR TITLE
Feature/styled components and react native elements

### PR DESCRIPTION
**Pull request by SoleCincis**
Setup of styled-components and react-native-elements

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,9 +19,12 @@
   },
   "dependencies": {
     "expo": "^32.0.0",
+    "node": "^12.2.0",
```
Comment (SoleCincis) [inline package.json]: this is creating a conflict 
Screenshot 2019-05-17 at 16 32 51

Comment (donni106) [reply package.json]: Ok. Please remove the node package from the projects package.json. Node should be used from your global system packages.

Comment (donni106) [reply package.json]: Ok. Please remove the node package from the projects package.json. Node should be used from your global system packages.

File: src/config/texts.js [hunk 2]
```diff
@@ -14,3 +14,12 @@ export const texts = {
     policy: 'Datenschutz'
   }
 };
+
+import styled, { css } from 'styled-components/native';
+import { colors } from './colors';
+
+export const ScreenTitle = styled.Text`
```
Comment (donni106) [inline src/config/texts.js]: This has to be removed here and put somewhere in /components/. This is a styled `Text` component and not part of text configs.

File: package.json (hunk 1 above)
Comment (donni106) [reply package.json]: Resolved with cherry picking the [commit](https://github.com/ikuseiGmbH/smart-village-app-app/commit/cc2880c3fe49f4f1a374df8c3aaebd144d65b4fa) from branch: **feauture/working-on-components**

File: src/config/texts.js (hunk 2 above)
Comment (SoleCincis) [reply src/config/texts.js]: did it in new branch 'workin-on-components'

**Pull request merged**